src/person_linkage_case_study_utils/snakemake_utils.py

```diff
@@ -127,4 +127,6 @@ def get_directory_wrapper_if_necessary(papermill_params):
 # because it does not generate incremental output, nor output notebooks when there is
 # an error. See https://github.com/snakemake/snakemake/pull/2857
 def dict_to_papermill(d):
-    return " ".join([f"-p {k} {v}" for k, v in d.items()])
+    from shlex import quote
```

Comment:
nit--do you have some reason for not wanting to put this import at the top?

Reply:
Nope!